tools/openblas_support.py

```diff
@@ -51,18 +52,32 @@ def download_openblas(target, arch, ilp64):
         suffix = 'macosx_10_9_x86_64-gf_1becaaa.tar.gz'
         filename = f'{ANACONDA}/{OPENBLAS_LONG}/download/openblas{fnsuffix}-{OPENBLAS_LONG}-{suffix}'
         typ = 'tar.gz'
+        if fnsuffix == "64_":
+            md5_expected = '17b028d96516ff4b61f9ce1bd6a634eb'
```

Comment:
Can we use sha256 instead? And maybe put these to a dictionary at the top of the file?